libcodechecker/server/config_database.py

```diff
@@ -4,41 +4,37 @@
 #   License. See LICENSE.TXT for details.
 # -------------------------------------------------------------------------
 """
-Database server handling.
+Database server handling for the server's configuration database.
```

Comment:
However, I think, it would make debugging harder. I ran into a situation while creating this patch where it was very useful that in the stack trace it was obvious which handler was used in a certain case.

If we were only passing by arguments, this information would be lost. I think there is a divide between us about this, because I am very much **for** keeping these things separate. (Originally, they were even in separate folders under `libcodechecker.server` but I did not want to have import strings that contain 3-4 elements in their path.)